django_browserid/views.py

```diff
@@ -6,7 +6,7 @@
 from django.conf import settings
 from django.contrib import auth
 from django.http import HttpResponse
-from django.template import RequestContext
+from django.template import RequestContext, Template
```

Comment:
What happens if I configure a Jinja backend as my default and only template engine, and I'm using a library like `session_csrf` that uses a different CSRF token from the default CSRF context processor? Will `RequestContext` still pick up the right CSRF token?

If not, then I think we need to generate a template object from the default configured engine instead of directly importing Django's `Template` class.